trieste/acquisition/rule.py

```diff
@@ -611,22 +611,22 @@ class DiscreteThompsonSampling(AcquisitionRule[TensorType, SearchSpace]):
 
     The model is sampled either exactly (with an :math:`O(N^3)` complexity), or sampled
     approximately through a random Fourier `M` feature decompisition
-    (with an :math:`O(\min(n^3,M^3))` complexity for a model trained on `n` points).
+    (with an :math:`O(\min(n^3,M^3))` complexity for a model trained on `n` points). The number
+    `M` of Fourier features is specified when building the model.
```

Comment:
Why does the number of features now live on the model rather than in the rule/acquisition function?

Reply:
This is open question about how we want to be able to define what sort of trajectory sampler we package with a model. I thought that defining this when we build the model would be a nice way to do this, e.g. if we included other types of fourier features at some point then we could tell the model which ones it will use. Any thoughts @vpicheny @sebastianober ?

Reply:
@henrymoss we decided against this right? i think we can resolve

Reply:
this is also how I coded it in my quantile experiment